common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4830,6 +4830,11 @@
           "<variableName> is a VARIABLE and cannot be updated using the SET statement. Use SET VARIABLE <variableName> = ... instead."
         ]
       },
+      "SQL_SCRIPTING_NOT_ENABLED" : {
+        "message" : [
+          "SQL scripting is under development and not all features are supported. To enable existing features set <sqlScriptingEnabled> to `true`."
```

Comment:
It is obvious that the feature is not supported since this is a sub-class of `UNSUPPORTED_FEATURE`. Would be better just shortly describe the feature itself.

Reply:
Is this ok:

```
"SQL Scripting is under development and not all features are supported. SQL Scripting enables users to write procedural SQL including control flow and error handling. To enable existing features set <sqlScriptingEnabled> to `true`."
```

Reply:
SGTM

Reply:
Done.